romancal/resample/l3_wcs.py

```diff
@@ -58,34 +60,22 @@ def assign_l3_wcs(model, wcs):
         l3_wcsinfo.dec_corn4 = footprint[3][1]
         l3_wcsinfo.s_region = compute_s_region_keyword(footprint)
 
-    # Fill out wcs-general information
-    try:
-        l3_wcsinfo.x_ref = -transform["crpix1"].offset.value
-        l3_wcsinfo.y_ref = -transform["crpix2"].offset.value
-    except IndexError:
-        log.warning(
-            "WCS has no clear reference pixel defined by crpix1/crpix2. Assuming reference pixel is center."
-        )
-        l3_wcsinfo.x_ref = pixel_center[0]
-        l3_wcsinfo.y_ref = pixel_center[1]
```

Comment:
The "except" blocks here were conceptually trying to deal with the case that someone passes a non-tangent-plane WCS / non-FITSWCS WCS to the L3 code as an external WCS.  The idea was that we could still populate some kinds of useful reference pixel / scale information for generic WCSes, even when this information isn't enough to completely reproduce the input WCS object.  I think I'd prefer not to lose that functionality; do you see this differently?

Reply:
Added them back.